lib/exgboost/training/state.ex

```diff
@@ -3,9 +3,18 @@ defmodule EXGBoost.Training.State do
   @enforce_keys [:booster]
   defstruct [
     :booster,
-    meta_vars: %{},
     iteration: 0,
     max_iteration: -1,
-    metrics: %{}
+    meta_vars: %{},
+    metrics: %{},
```

Comment:
Im trying to remember why I didnt just use the `meta_vars` to hold the metrics in...